### DISCUSSION TITLE
Error Downloading LFS Object in [ForestFireDetection]

### DISCUSSION THREAD

**Discussion by zz39** [akdasUAF/ForestFireDetection#32] [Q&A] [answered]
Hi @YaliWang2019.  I'm trying to clone the [repository name] repository to my local machine, but I encountered an error while downloading the file ```Models/weights/dbn_1_RBM.joblib``` (199 MB).

**Error Message:**
```
Downloading Models/weights/dbn_1_RBM.joblib (199 MB)
Error downloading object: Models/weights/dbn_1_RBM.joblib (f25e384): Smudge error: Error downloading Models/weights/dbn_1_RBM.joblib (f25e384d7f77efcbb8fd1d44c538bb48534f48a797d93992728b1ff52849193b): batch response: This repository is over its data quota. **Account responsible for LFS bandwidth should purchase more data packs to restore access.**
```
**Issue:**

It appears the repository has exceeded its Git LFS data quota, preventing the download of large files like dbn_1_RBM.joblib. This file seems to be crucial for running the app.py script. 

**Possible Solutions:**

Is it possible to increase the data quota or provide alternative access to the LFS file downloads?

Thanks in advance for your help!

**Comment by YaliWang2019** [comment, marked as answer]
Yeah, uploading large files to GitHub is tricky. I remember spending quite a long time figuring out how to use their LFS to upload those model files...
I'll try to fix that as soon as possible and I'll let you know once I get it fixed : )
(Probably I'll try just put that file in a Google Drive XD

**Reply by zz39** [reply]
Thank you for this super quick reply, Yali! Yes - I guess the straightforward way is just using Google Drive or other cloud storage...at least they are good for backup when Git LFS not working.

**Reply by zz39** [reply]
Hi @YaliWang2019,

Just checking in to see if you've had a chance to tweak the LFS file(dbn_1_RBM.joblib) or if you've managed to upload it to Google Drive. Looking forward to your update :D

**Reply by YaliWang2019** [reply]
Hi! Sorry for the late response. I’ve added the three DBN models in the same folder in the Readme file, please check the new instructions in the Readme :D

**Reply by zz39** [reply]
Got the three DBN models. Now I can get the app.py running. Thanks a lot for the update!

**Reply by YaliWang2019** [reply]
No problem! If anything goes wrong please feel free to let me know ; )
